src/features/match/service/readAllMatch.ts

```diff
@@ -28,7 +29,7 @@ const readAllMatch = async () => {
 
 export const useMatchList = () => {
     return useQuery({
-        queryKey: ["matchList"],
+        queryKey: [MATCH_QUERY_KEY_FACTORY.READ_ALL_MATCH],
```

Comment:
`READ_ALL_MATCH` 는 `() => ["MATCH"]` 함수 자체를 리턴하기에,
```suggestion
        queryKey: [MATCH_QUERY_KEY_FACTORY.READ_ALL_MATCH()]
```
로 수정하는것이 올바르게 동작할 것 같습니다

Reply:
추후 `queryInvalidation` 시에도 위 코드처럼 함수 자체를 넘겨주는것이 아닌, 함수를 호출하는 코드로 변경해주시면 좋을것 같아요!